### PR TITLE
MEN-3762 chmod 600 on mender.conf

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: mender-convert-modify [hunk 1]
```diff
@@ -193,6 +193,7 @@ cat <<- EOF > work/mender.conf.data
 EOF
 
 run_and_log_cmd "sudo cp work/mender.conf.data work/rootfs/data/mender/mender.conf"
+run_and_log_cmd "chmod 600 work/rootfs/data/mender/mender.conf"
```
Comment (kacf) [inline mender-convert-modify:196]: I think you need `sudo` here.

Comment (merlin-northern) [reply mender-convert-modify]: > I think you need `sudo` here.

ah, crap, copy-pasted from the line above, and removed sudo by mistake. [thanks](https://github.com/mendersoftware/mender-convert/pull/219/commits/564d263f53be0cc4b0b69e3f45f6153541462ae3)

**Comment by merlin-northern** [comment]
> Thanks!
> 
> I realized now though, that this only changes one of the two `mender.conf` files. This is the one in `/data`, you also have [several different variants in `/etc`](https://github.com/mendersoftware/mender-convert/search?q=%22%2Fetc%2Fmender%2Fmender.conf%22&unscoped_q=%22%2Fetc%2Fmender%2Fmender.conf%22).

yeah I checked those. but it is redirection, it should not change the existing conf file mode. or do you think it is possible that those redirections create the file?

**Comment by kacf** [comment]
They are definitely creating the file, since the input image is an image without Mender.

**Comment by merlin-northern** [comment]
> They are definitely creating the file, since the input image is an image without Mender.

ok! [fixed](https://github.com/mendersoftware/mender-convert/pull/219/commits/198e22767a7410c96eda0ba07cf605daf2224c5c)

**Pull request merged**